docs/reference-guide/index.md

```diff
@@ -0,0 +1,27 @@
+---
+myst:
+  html_meta:
+    "description": "Plone Reference guide"
+    "property=og:description": "Plone Reference guide"
+    "property=og:title": "Reference guide"
+    "keywords": "Plone 6, reference guide"
+---
+
+# Reference guide
+
+This reference guide details functions, modules, and objects included in Plone, describing what they are and what they do. For learning how to use Plone, see the [complete documentation](https://6.docs.plone.org/index.html).
+
+```{note}
+This part of the documentation is under construction.
+You can find documentation for plone in various locations through the [search feature](https://6.docs.plone.org/search.html?q=plone).
+You can help consolidate all of reference material here, even if it is to let us know what is missing, by [creating an issue](https://github.com/plone/documentation/issues/new?assignees=&labels=&projects=&template=new-issue-form.yml).
+```
```

Comment:
We've tried using "under construction", TODOs, and other things to encourage contributions to documentation, none of which have been effective.

I just created an issue out of disgust and embarrassment https://github.com/plone/documentation/issues/1882.

```suggestion
```